libs/ui/widgets/src/lib/ui-widgets.module.ts

```diff
@@ -12,11 +12,13 @@ import { MatProgressSpinnerModule } from '@angular/material/progress-spinner'
 import { MatIconModule } from '@angular/material/icon'
 import { SpinningLoaderComponent } from './spinning-loader/spinning-loader.component'
 import { CommonModule } from '@angular/common'
+import { PopoverComponent } from './popover/popover.component'
 
 @NgModule({
   declarations: [
     ColorScaleComponent,
     ProgressBarComponent,
+    PopoverComponent,
```

Comment:
The `PopoverComponent` is missing in the exports Array otherwise it cannot be used by the metadata-quality component.